tasks/configuration.yml

```diff
@@ -1,7 +1,21 @@
 ---
 
+- name: wrap /etc/issue message in cowsay
+  ansible.builtin.shell:
+    cmd: "/usr/games/cowsay"
+    stdin: "{{ motd_issue_message }}"
+  register: motd_issue_cowsay_message
+  when: motd_cowsay is true
+
+- name: wrap /etc/motd message in cowsay
+  ansible.builtin.shell:
+    cmd: "/usr/games/cowsay"
+    stdin: "{{ motd_message }}"
```

Comment:
```suggestion
    stdin: "{{ motd_message }}"
  changed_when: false
```

Reply:
Resolved in [47832d3](https://github.com/adfinis/ansible-role-motd/pull/5/commits/47832d3b079677522019d743d7c61d6c355ebf28)